example/cppapi/websocket/WSBuffer.cc

```diff
@@ -157,7 +157,7 @@ WSBuffer::read_buffered_message(std::string &message, int &code)
 std::string
 WSBuffer::ws_digest(std::string const &key)
 {
-#if OPENSSL_VERSION_NUMBER < 0x10100000L
+#if OPENSSL_VERSION_NUMBER < 0x10100000L || defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
Do we really need all this ifdeffery? Can we just unconditionally use the modern API?